### PR TITLE
[testnet] migration proposal

### PR DESCRIPTION
**Pull request by lightmark**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by sherry-x** [comment]
you need to have this on 1.12 branch

File: aptos-move/aptos-release-builder/data/proposals/create_apt_mapping.move [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+script {
+    use aptos_framework::aptos_governance;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(
+            proposal_id,
+            @0x1,
+            { { script_hash } },
+        );
+        aptos_framework::coin::create_coin_conversion_map(&framework_signer);
```
Comment (movekevin) [inline aptos-move/aptos-release-builder/data/proposals/create_apt_mapping.move:13]: nit: Can you import coin and use coin::function_name

**Review by movekevin — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/coin_to_fa_migration.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+remote_endpoint: ~
+name: coin_to_fa_migration
+proposals:
+  - name: enable_coin_to_fa_migration
+    metadata:
+      title: "Enable the mapping from coin to fungible asset including APT"
+      description: "Enables the changes in AIP-63 https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-63.md"
+      source_code_url: ""
```
Comment (sherry-x) [inline aptos-move/aptos-release-builder/data/coin_to_fa_migration.yaml:9]: this won't work, you need to put a real url

**Review by sherry-x — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/proposals/create_apt_mapping.move [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+script {
+    use aptos_framework::aptos_governance;
+    use aptos_framework::coin;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(
+            proposal_id,
+            @0x1,
+            { { script_hash } },
```
Comment (sherry-x) [inline aptos-move/aptos-release-builder/data/proposals/create_apt_mapping.move:12]: where does this coming from?

**Pull request merged**